tests/ui/postgres/gated/chrono.stderr

```diff
@@ -1,60 +1,60 @@
-error: optional feature `chrono` required for type DATE of column #1 ("now")
+error: optional feature `time` required for type DATE of column #1 ("now")
```

Comment:
This is weird, these shouldn't have changed. @jplatte what command are you running `trybuild` tests with?

Reply:
I removed `#[ignored]` from the ui test, started the `postgres_12` docker image from `docker-compose.yml`, set the `DATABASE_URL` like in the CI config and ran `TRYBUILD=overwrite cargo test --features runtime-async-std-rustls,postgres`.

Reply:
I think I may have used a different postgres version before, otherwise I have no clue why this changed only now.

Reply:
ping @abonander Should I revert this change?

Reply:
no, the tests seem to pass so I'll merge it